package-lock.json

```diff
@@ -0,0 +1,4663 @@
+{
```

Comment:
I don't think this file is supposed to be added to the project. yarn.lock supersedes package-lock.json. are you building with npm or yarn?